### PR TITLE
refactored master to controlPlane

### PR DESCRIPTION
**Pull request by melvinj98**
#### What type of PR is this?
/kind cleanup
/wg naming
/sig testing
/area test

#### What this PR does / why we need it:
Refactor master to controlPlane in stats.go and kubelet_perf.go

#### Which issue(s) this PR fixes:
Fixes #

#### Special notes for your reviewer:
References issue #94901

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
```docs

```

### CODE REVIEW

File: test/e2e/framework/kubelet/stats.go [hunk 1]
```diff
@@ -551,11 +551,11 @@ func (r *ResourceMonitor) GetLatest() (ResourceUsagePerNode, error) {
 // GetMasterNodeLatest returns the latest resource usage of master and node.
 func (r *ResourceMonitor) GetMasterNodeLatest(usagePerNode ResourceUsagePerNode) ResourceUsagePerNode {
 	result := make(ResourceUsagePerNode)
-	var masterUsage ResourceUsagePerContainer
+	var controlPlaneUsage ResourceUsagePerContainer
 	var nodesUsage []ResourceUsagePerContainer
```
Comment (mgutierrez98) [inline test/e2e/framework/kubelet/stats.go:555]: `workerNodesUsage` may be a better fit for this, thoughts?

File: test/e2e/framework/kubelet/stats.go [hunk 2]
```diff
@@ -643,11 +643,11 @@ func (r *ResourceMonitor) GetCPUSummary() NodesCPUSummary {
 // GetMasterNodeCPUSummary returns summary of master node CPUs.
 func (r *ResourceMonitor) GetMasterNodeCPUSummary(summaryPerNode NodesCPUSummary) NodesCPUSummary {
 	result := make(NodesCPUSummary)
-	var masterSummary ContainersCPUSummary
+	var controlPlaneSummary ContainersCPUSummary
 	var nodesSummaries []ContainersCPUSummary
```
Comment (mgutierrez98) [inline test/e2e/framework/kubelet/stats.go:647]: `workerNodesSummaries` here as well

File: test/e2e/framework/kubelet/stats.go [hunk 3]
```diff
@@ -551,11 +551,11 @@ func (r *ResourceMonitor) GetLatest() (ResourceUsagePerNode, error) {
...
 	result := make(ResourceUsagePerNode)
-	var masterUsage ResourceUsagePerContainer
+	var controlPlaneUsage ResourceUsagePerContainer
 	var nodesUsage []ResourceUsagePerContainer
 	for node, usage := range usagePerNode {
 		if strings.HasSuffix(node, "master") {
-			masterUsage = usage
+			controlPlaneUsage = usage
 		} else {
 			nodesUsage = append(nodesUsage, usage)
```
Comment (mgutierrez98) [inline test/e2e/framework/kubelet/stats.go:560]: for consistency with above change

File: test/e2e/framework/kubelet/stats.go [hunk 4]
```diff
@@ -643,11 +643,11 @@ func (r *ResourceMonitor) GetCPUSummary() NodesCPUSummary {
...
 	result := make(NodesCPUSummary)
-	var masterSummary ContainersCPUSummary
+	var controlPlaneSummary ContainersCPUSummary
 	var nodesSummaries []ContainersCPUSummary
 	for node, summary := range summaryPerNode {
 		if strings.HasSuffix(node, "master") {
-			masterSummary = summary
+			controlPlaneSummary = summary
 		} else {
 			nodesSummaries = append(nodesSummaries, summary)
```
Comment (mgutierrez98) [inline test/e2e/framework/kubelet/stats.go:652]: consistency with above change

**Comment by mgutierrez98** [comment]
Would love some feedback on these possible changes when you have the time

File: test/e2e/framework/kubelet/stats.go [hunk 5]
```diff
@@ -551,11 +551,11 @@ func (r *ResourceMonitor) GetLatest() (ResourceUsagePerNode, error) {
 // GetMasterNodeLatest returns the latest resource usage of master and node.
 func (r *ResourceMonitor) GetMasterNodeLatest(usagePerNode ResourceUsagePerNode) ResourceUsagePerNode {
```
Comment (oomichi) [inline test/e2e/framework/kubelet/stats.go:552]: Why don't you rename this function?

**Review by oomichi — COMMENTED** [review]
/ok-to-test

File: test/e2e/framework/kubelet/stats.go (hunk 5 above)
Comment (melvinj98) [reply test/e2e/framework/kubelet/stats.go:552]: I wasn't sure where this function was being referenced so I decided to not touch it until I'm sure.

Comment (oomichi) [reply test/e2e/framework/kubelet/stats.go:552]: The function is called from https://github.com/kubernetes/kubernetes/blob/b47f8263e18c7b13dba33fba23187e5e0477cdbd/test/e2e/node/kubelet_perf.go#L112 only according to https://cs.k8s.io/?q=GetMasterNodeLatest&i=nope&files=&excludeFiles=&repos=

File: test/e2e/node/kubelet_perf.go [hunk 6]
```diff
@@ -1,291 +0,0 @@
-/*
```
Comment (oomichi) [inline test/e2e/node/kubelet_perf.go:1]: Why do we need to delete this kubelet_perf.go file?
I cannot find the reason from the pull request message.

Comment (melvinj98) [reply test/e2e/node/kubelet_perf.go:1]: I mistakenly deleted it while attempting to figure out why the tests were failing. Would you be able to offer me some guidance on how I could fix these errors?

Comment (oomichi) [reply test/e2e/node/kubelet_perf.go]: Ah you have already fixed it by yourself.
Thanks.

**Comment by melvinj98** [comment]
As this is my first PR, I'm not entirely sure about the procedure. Is there anything else that I need to do from my end?

**Comment by mmiranda96** [comment]
It looks like tests are failing. Let's try rerun them.
/retest

**Comment by melvinj98** [comment]
I've tried looking for reasons that could be causing these tests to fail but I haven't been able to find anything

**Comment by gjkim42** [comment]
> I've tried looking for reasons that could be causing these tests to fail but I haven't been able to find anything

I think those tests are broken. We don't necessarily need to pass those tests to merge this PR(they are not required tests)
You only need lgtm label from the reviewer.

**Comment by mgutierrez98** [comment]
Any updates regarding this PR?

File: test/e2e/framework/kubelet/stats.go [hunk 7]
```diff
@@ -640,14 +640,14 @@ func (r *ResourceMonitor) GetCPUSummary() NodesCPUSummary {
...
-// GetMasterNodeCPUSummary returns summary of master node CPUs.
-func (r *ResourceMonitor) GetMasterNodeCPUSummary(summaryPerNode NodesCPUSummary) NodesCPUSummary {
+// GetControlPlaneCPUSummary returns summary of control plane CPUs.
+func (r *ResourceMonitor) GetControlPlaneCPUSummary(summaryPerNode NodesCPUSummary) NodesCPUSummary {
 	result := make(NodesCPUSummary)
-	var masterSummary ContainersCPUSummary
+	var controlPlaneSummary ContainersCPUSummary
 	var nodesSummaries []ContainersCPUSummary
 	for node, summary := range summaryPerNode {
 		if strings.HasSuffix(node, "master") {
```
Comment (ehashman) [inline test/e2e/framework/kubelet/stats.go:649]: It seems we're not actually changing the node name in this PR? We will need to do that, won't we?

**Review by ehashman — COMMENTED** [review]
/test pull-kubernetes-node-kubelet-serial-containerd
/test pull-kubernetes-node-kubelet-serial

**Comment by pacoxu** [comment]
/label tide/merge-method-squash

> /remove-approve
/approve cancel

**Pull request closed without merging**